JSDemos/Demos/Popup/Overview/Angular/description.md

```diff
@@ -1,10 +1,34 @@
+
+- On outside click    
+Users can also hide the Popup on outside clicks, if you enable the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#hideOnOutsideClick) property.
+
+## Configure the Popup
+
+The Popup inner area is divided into two parts: title area and content area.
+
+- Content area       
+To populate the Popup with content, add markup inside the component. You can also enable a toolbar at the bottom of the content area. To do this, declare [toolbarItems](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/) in the markup as shown in this demo. Set each item's [toolbar](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#toolbar) property to `bottom`. To learn more about toolbar configuration, refer to the following tutorial: [Getting Started with Toolbar](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/)
+
+- Title area       
+Set [showTitle](Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) to `true` and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to specify the caption. You can also add a toolbar to the title area. Add [toolbarItems](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/) markup and set each item's [toolbar](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#toolbar) property to `top`. If you want to display an item in the overflow menu, as shown in this demo, set the item's [locateInMenu](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#locateInMenu) to `always`.
+
+## Resize and Position
+
+To specify the Popup's size, use the [height](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#height) and [width](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#width) properties.
+
+In this demo, each Popup's location is relative to the image. The code sets the **my**, **at**, and **of** properties of the [position](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#position) object. The configuration in the demo reads as follows: "place **my** `center` side **at** the `bottom` side **of** the `#image${employee.ID}` element."
+
+Use the [container](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#container) property to specify the container to render the Popup in. If you set the [container](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#container) property to a specific element on the page, the [shading](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#shading) applies to this element.
```

Comment:
```suggestion
Use the [container](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#container) property to specify in which container to render the Popup. If you set the [container](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#container) property to a specific element on the page, [shading](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#shading) applies to this element.
```

Reply:
I tried to avoid 'which' here and you added it back 😄 Is my version really that bad?

Reply:
Your version was an incomplete sentence. We can say this instead:
```suggestion
Use the [container](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#container) property to select the container in which you want to render the Popup. If you set the [container](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#container) property to a specific element [WHAT ELEMENT?] on the page, the [shading](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#shading) applies to this element.
```